git-cliff/Cargo.toml

```diff
@@ -23,7 +23,9 @@ path = "src/bin/mangen.rs"
 
 [features]
 # check for new versions
-default = ["update-informer", "github", "gitlab", "gitea", "bitbucket"]
+default = ["update-informer", "integrations"]
+# enable all integrations
+integrations = ["github", "gitlab", "gitea", "bitbucket"]
```

Comment:
I understand where you coming from but I still believe that the same can be achieved via disabling the `remote` feature flag (because the other feature flags depend on it already). But maybe I'm wrong, can you test this?

Reply:
Ah I got it now. I thought you were doing the reverse.